src/rest.ts

```diff
@@ -39,18 +39,29 @@ export function getTDPDatabases(): Promise<IDatabaseDesc[]> {
   return getAPIJSON(`${REST_DB_NAMESPACE}/`);
 }
 
-export interface IViewDesc {
+export interface IViewDescription {
+
+  /**
+   * idType of the DBView, can be null
+   */
+  idType: string | null;
```

Comment:
I think we should also add the type `IDType` here, so it should be `idType: IDType | string | null;`. the reason for this is that we might also want to use the actual resolved IDType in the frontend and not only the string (e.g. `GeneSymbol_human`).

Reply:
or the shorter version would be `idType: IDTypeLike | null` --> and I'm not sure if we need the `| null`

Reply:
it's only needed if you have `--strictNullChecks`, but I thought I include it because I already know it can be null.

Reply:
Set it to `idType: IDTypeLike | null`.